Components/NavigationBurger/functions.php

```diff
@@ -14,7 +14,7 @@
 add_filter('Flynt/addComponentData?name=NavigationBurger', function ($data) {
     $data['menu'] = new Timber\Menu('navigation_burger');
     $data['logo'] = [
-        'src' => get_theme_mod('custom_header_logo') ? get_theme_mod('custom_header_logo') : Asset::requireUrl('Components/NavigationBurger/Assets/logo.svg'),
+        'src' => get_theme_mod('custom_header_logo') ? get_theme_mod('custom_header_logo') : Asset::requireUrl('../assets/images/logo.svg'),
```

Comment:
Do we need the `../` relative path? I expected the folder being on the same level as `Components`.